docs/star-tool.md

```diff
@@ -0,0 +1,226 @@
+# The `star` tool
+
+The `star` tool is a cross-platform command-line tool, distributed as a stand-alone console application, used to manage Starcounter databases and applications. By using the `star` tool, you can – for example – create, modify, delete and query Starcounter databases, as well as import and export data from compatible 3:rd party data sources.
+
+The easiest way to explore the features of the `star` tool is to use it, in particular with the `--help` option, which prints help information about available commands and options. In this article, we will provide a reference for the available commands and describe some common use cases.
+
+## Distribution
+
+The Windows and Linux versions of the tool are downloaded together with the main [Starcounter binaries](README.md#installation). For Linux operating systems, make sure to also install the prerequisites listed at the [installation page](README.md#installation) and grant execution permissions (`chmod -R u+x /path/to/star/tool/folder`).
+
+## Usage
+
+The `star` tool is a console application with a command-line interface, and we interact with it using commands, options and arguments.
+
+The global command pattern is:
+
+```
+star [options] [command]
+```
+
+Specific commands may have different sub-patterns, which are defined in their respective sections below. The option `--help` is defined for each command, and can be used to print details on available options and sub-commands.
+
+In examples below, Windows-style directory paths are used. On Linux operating systems, the `star` tool expects Linux-style paths.
+
+### Commands
+
+The available commands can be split into four categories:
+
+- [Database management commands](#database-management), used for creating and starting databases.
+- [Import and export commands](#import-and-export), used for importing and exporting data to or from SQLite database files.
+- [Information commands](#information), e.g. printing all tables or listing all indexes of a given table.
+- [SQL REPL](#sql-repl), used for executing DML and DDL statements manipulating the database schema and data.
+
+#### Database management
+
+##### `new`
+
+```
+star new [options] <path>
+```
+
+The `new` command creates a new database at the absolute or relative path provided in the `<path>` argument. If the directory referenced by the path does not exist, it will be created. To create a new database at the path `C:\databases\mydb`, we run:
+
+```
+star new C:\databases\mydb
+```
+
+##### `start`
+
+```
+star start [options] <db>
+```
+
+The `start` command starts the database referenced by the absolute or relative path given in the `<db>` argument. To start an existing database located at `C:\databases\mydb`, we run:
+
+```
+star start C:\databases\mydb
+```
+
+#### Import and export
+
+##### `reload`
```

Comment:
In this case, the reason is trivial: they’re “import and export” commands, hence first reload then unload. But I guess the order is not that important, I can change it

Reply:
> In this case, the reason is trivial: they’re “import and export” commands, hence first reload then unload. But I guess the order is not that important, I can change it

Just figured it's hard to import (reload) something you haven't exported (unloaded) yet. Especially so since we state that we can only ever import data files exported by us. 😎 

But not important, do as you want.